site-packages/integralstor/remote_monitoring.py

```diff
@@ -0,0 +1,179 @@
+import os
+import shutil
+import urllib
+import json
+from integralstor_utils import config, networking
+
+
+def _get_config_file_path():
+    server_config_file_path = None
+    try:
+        config_files_dir, err = config.get_config_files_dir()
+        if err:
+            raise Exception(err)
+        server_config_file_path = '%s/remote_monitoring_servers' % config_files_dir
+        if not os.path.isfile(server_config_file_path):
+            if not os.path.exists(config_files_dir):
+                os.makedirs(config_files_dir)
+            with open(server_config_file_path, 'w') as f:
```

Comment:
True.. Was thinking about this. Problem is that every calling function will now have to call two functions then - one to get the path and another to initialize the file if it does not exist. Since this is being called from two places, it would be duplication of code on the calling side. The other option is to rename this to _get_and_initialize_config_file_path() or something like that. What do you think?